src/uk/gov/hmcts/contino/azure/Acr.groovy

```diff
@@ -76,7 +79,8 @@ class Acr extends Az {
    *   stdout of the step
    */
   def build(DockerImage dockerImage, String additionalArgs) {
-    this.az "acr build --no-format -r ${registryName} -t ${dockerImage.getBaseShortName()} --subscription ${registrySubscription} -g ${resourceGroup} --build-arg REGISTRY_NAME=${registryName}${additionalArgs} ."
+    def buildTag = projectBranch.isPR() ? dockerImage.getBaseTaggedName() : dockerImage.getBaseShortName()
```

Comment:
why this change? it seems like this uses the full registry name on a PR?

Reply:
I.. honestly have no idea, most of the changes are for the purposes of building on preview as an actual branch..
would this help or affect that adversely? 
I don't see why it would

Reply:
exactly, this is my last concern atm, could you try reverting it and we can see if it works for you?

Reply:
Okay I have reverted that change, test seem to be passing.
When I try deploy the web-apps to preview from master, we will see if that did anything before 🤔